src/main/java/controller/LadderController.java

```diff
@@ -0,0 +1,41 @@
+package controller;
+
+import domain.LadderGame;
+import domain.ladder.LadderHeight;
+import domain.ladder.Ladder;
+import domain.LadderResult;
+import domain.ladder.LadderWidth;
+import view.InputView;
+import view.OutputView;
+
+import java.util.List;
+
+public class LadderController {
+
+    public void run(){
+        List<String> players = InputView.inputPlayerNames();
+        List<String> results = InputView.inputGamePrice();
+        LadderWidth width = new LadderWidth(players.size());
+        LadderHeight height = new LadderHeight(InputView.inputLadderHeight());
+
+        Ladder ladder = new Ladder(width, height);
+        LadderGame ladderGame = new LadderGame(ladder, results);
+        LadderResult ladderResult = ladderGame.playGame();
+        List<String> gameResult = ladderResult.getResults();
+
+        OutputView.printLadder(ladder, players, results);
+
+        while (true){
+            String who = InputView.inputWhoseResult();
+
+            if (who.equals("exit")){
+                OutputView.printExitMessage();
+                break;
+            } else if (who.equals("all")){
+                OutputView.printAllResult(gameResult, players);
+                continue;
+            }
+            OutputView.printIndividualResult(gameResult,players, who);
```

Comment:
```suggestion
            if (who.equals("exit")){
                OutputView.printExitMessage();
                break;
            }
            
            if (who.equals("all")){
                OutputView.printAllResult(gameResult, players);
                continue;
            }
            OutputView.printIndividualResult(gameResult,players, who);
```

첫번째 if 조건에 break이 있어서 `else if`는 제거 할 수 있을 것 같네요!
요구사항에 "else를 사용하지 않는다"가 있는데요. `else if`도 사용 하지 않아야 합니다.

> 이 요구사항은 왜 있을까요? (답하지 않으셔도 괜찮아요)

Reply:
해당 요구사항이 왜 있는지 궁금합니다! 찾아봤을때는 코드의 복잡성이 증가하고, 코드의 가독성이 저하되어 디버깅에 어려움이 생긴다고 하는데 아직까지는 코드가 간단해서 그런지 오히려 더 복잡해진다고 느껴서요!

Reply:
좋은 의견입니다!
요구사항의 목적은 (제 생각으로는) 분기문을 최소화 해서 복잡성을 줄이고, 코드를 읽는 사람의 가독성을 높이기 위함이라 생각했어요. 

말씀대로 간단한 로직의 경우, 의무적으로 else를 없애는 것이 더 복잡하다고 느껴진다면 적용하지 않으셔도 됩니다!
목적에 맞게 사용한다면 문제는 없을 것 같네요.